client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx

```diff
@@ -61,7 +61,14 @@ const domainInputFieldIcon = ( isValidDomain: boolean, shouldReportError: boolea
 };
 
 const DomainPrice = ( { rawPrice, saleCost, currencyCode = 'USD' }: DomainPriceProps ) => {
-	const { __ } = useI18n();
+	const { __, hasTranslation } = useI18n();
+	const isEnglishLocale = useIsEnglishLocale();
+	const domainPriceText =
+		isEnglishLocale || hasTranslation( "We're paying" ) ? (
+			<div className="domains__domain-price-text">{ __( "We're paying" ) }&#8230;</div>
```

Comment:
Added ellipsis HTML entity separately to address i18n-ellipsis lint rule and moved it outside of the translation function since we don't need to translate the ellipsis.